### PR TITLE
Update minimum Flutter and Dart SDK versions

### PR DESCRIPTION
**Pull request by desistefanova**

### CODE REVIEW

**Comment by desistefanova** [comment]
@blagoev , @nielsenko , @bpmutter Can you review this change in the docs, please?

File: source/sdk/flutter/install.txt [hunk 1]
```diff
@@ -15,13 +15,13 @@ Prerequisites
 -------------
 
 - `Install Flutter in your development environment <https://docs.flutter.dev/get-started/install>`__.
-  You must use Flutter version 3.0 or later and Dart version 2.17.0 or later.
+  You must use Flutter version 3.0.2 or later and Dart version 2.17.5 or later.
```
Comment (mongodben) [inline source/sdk/flutter/install.txt:18]:
```suggestion
  To use the latest version of the SDK, you must use Flutter version 3.0.2 or later
  and Dart version 2.17.5 or later.
```

**Review by mongodben — APPROVED** [review]
lgtm with 1 suggestion for clarity

**Comment by mongodben** [comment]
> @blagoev , @nielsenko , @bpmutter Can you review this change in the docs, please?

also @bpmutter  is my personal github. my work one is @mongodben

**Comment by desistefanova** [comment]
To be aligned to [Update to analyzer 5+](https://github.com/realm/realm-dart/pull/1072)

File: source/sdk/flutter/install.txt [hunk 2]
```diff
@@ -15,13 +15,13 @@ Prerequisites
...
 - `Install Flutter in your development environment <https://docs.flutter.dev/get-started/install>`__.
-  You must use Flutter version 3.0 or later and Dart version 2.17.0 or later.
+  To use the latest version of the Realm SDK, you must use Flutter version 3.0.3 or later and Dart version 2.17.5 or later.
 
 .. warning:: Incompatible with Dart 2.17.3
 
    The Realm Flutter SDK and Realm Dart Standalone SDK are not compatible with
    Dart 2.17.3 due to an issue with the Dart virtual machine in this version.
-   You may use any other Dart 2.17.x version with Realm.
+   You may use Dart 2.17.5 version with Realm.
```
Comment (desistefanova) [inline source/sdk/flutter/install.txt:24]:
```suggestion
   You may use Dart 2.17.5 version or later with Realm.
```

**Pull request merged**